modules.d/90dmsquash-live/dmsquash-live-root.sh

```diff
@@ -33,19 +33,34 @@ overlay_size=$(getarg rd.live.overlay.size=)
 getargbool 0 rd.live.overlay.thin && thin_snapshot="yes"
 getargbool 0 rd.live.overlay.overlayfs && overlayfs="yes"
 
+# Take a path to a disk label and return the patent disk if it is a partition
+# Otherwise returns the original path
+function get_check_dev() {
+    dev_path="$(udevadm info -q path --name "$1")"
+    udevadm info -q property --path "${dev_path}" | grep -q 'DEVTYPE=partition' || { echo "$1"; return; }
+    parent="${dev_path%/*}"
+    udevadm info -q property --path "${parent}" | grep -q 'DEVTYPE=disk' || { echo "$1"; return; }
+    udevadm info -q property --path "${parent}" | grep -q 'ID_FS_TYPE=iso9660' || { echo "$1"; return; }
+
+    # Return the name of the parent disk device
+    udevadm info -q property --path "${parent}" | grep 'DEVNAME=' | sed 's/DEVNAME=//'
```

Comment:
Now the module should install `grep` and `sed` here:

https://github.com/dracutdevs/dracut/blob/fe8fa2b0cadbb33e27c8dd8b5851548dcd65835c/modules.d/90dmsquash-live/module-setup.sh#L25